src/views/Login.jsx

```diff
@@ -12,11 +17,21 @@ function Login() {
 		}
 	}, [user]);
 
+	const toggleDarkMode = () => {
+		setDarkMode(!darkMode);
+	};
+
 	return (
-		<div className="flex justify-center items-center h-screen">
+		<div
+			className={`${darkMode && 'dark'} flex justify-center items-center h-screen`}
+		>
 			<div className="bg-black text-white rounded-xl w-11/12 max-w-lg p-8 shadow-xl">
```

Comment:
```suggestion
			<div className="bg-transparent text-white  w-11/12 max-w-lg p-8 ">
```
This way the whole screen is the same color. We will need to adjust exactly which color to use based on the new style and make it Tailwind instead of a separate css file.

Reply:
Thanks Queralt all updated now!